scheduler/src/cook/kubernetes/controller.clj

```diff
@@ -195,15 +195,22 @@
         ; We always store the updated state, but only reprocess it if it is genuinely different.
         (swap! existing-state-map assoc pod-name new-state)
         (when-not (existing-state-equivalent? old-state new-state)
-          (process kcc pod-name))))))
+          (try
+            (process kcc pod-name)
+            (catch Exception e
+              (log/error e "Error while processing pod-update for" pod-name))))))))
```

Comment:
Can we include the compute cluster name in this log? To be consistent with other spots, let's do 

```
In compute cluster $name, error while processing pod update for ...
```